source/localizable/tutorial/autocomplete-component.md

```diff
@@ -15,7 +15,8 @@ As before, this creates a Handlebars template
 The Handlebars template looks like this:
 
 ```app/templates/components/filter-listing.hbs
-City: {{input value=filter key-up=(action 'autoComplete')}} 
+City: {{input value=filter key-up=(action 'autoComplete')}}
```

Comment:
Need to try this but I think you can just call it like this and remove the need for handling the action within the component.

```hbs
{{input value=filter key-up=(action autoComplete filter)}}
```